### PR TITLE
Fixed: Some typos and grammatical errors

### PR DESCRIPTION
**Pull request by HVbajoria**
Dear sir/ma'am,
In this pull request I tried manually searching through the files for some typos or grammatical mistakes that might not look good.
I have tried finding and correcting them.
Could you please review and help me improve if required.
Will be happy to learn.
Thank you
Happy HacktoberFest.

### CODE REVIEW

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your contribution @HVbajoria but a lot of your fixes actually interfere with our [style guide](https://www.notion.so/medusajs/Style-Guide-Docs-fad86dd1c5f84b48b145e959f36628e0) which is mentioned in our contribution guidelines.

I advise you to read it first, then make the changes.

**Comment by HVbajoria** [comment]
Hello @shahednasser ,
Sorry for the inconvenience.
I have checked the style guide and made the changes.
Could you please check once?
Thank you

File: docs/content/contribution-guidelines.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ npm install @medusajs/medusa-cli -g
 
 ## Linting with Vale
 
-Medusa uses Vale to lint documentation pages and perform checks on incoming PRs into the repository.
+Medusa uses Vale to lint documentation pages and performs checks on incoming PRs into the repository.
```
Comment (shahednasser) [inline docs/content/contribution-guidelines.md:138]: Undo this change

**Review by shahednasser — CHANGES_REQUESTED** [review]
I added some comments below. In addition, please undo all changes to files under `docs/services` as they're not maintained.

**Comment by HVbajoria** [comment]
Hello @shahednasser ma'am,
I have made the suggested changes.
Could you please check once and let me know if I should make any more improvements to make it good?
Will be happy to learn in the process.
Thank you

File: docs/content/contribution-guidelines.md [hunk 2]
```diff
@@ -69,17 +69,17 @@ When you add a new page to the documentation, you must add the new page in `www/
 
 ### Terminology
 
-When the documentation page is a conceptual or overview documentation, the label in the sidebar should start with a noun.
+When the documentation page is a conceptual or an overview documentation, the label in the sidebar should start with a noun.
 
-When the documentation page is a tutorial documentation, the label in the sidebar should start with a verb. An exception of this rule are integration documentations and upgrade guides.
+When the documentation page is tutorial documentation, the label in the sidebar should start with a verb Exceptions to this rule are integration documentation and upgrade guides.
```
Comment (shahednasser) [inline docs/content/contribution-guidelines.md:74]: Sorry this one is an error from my side, please change it to:

```suggestion
When the documentation page is tutorial documentation, the label in the sidebar should start with a verb. Exceptions to this rule are integration documentation and upgrade guides.
```

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by HVbajoria** [comment]
Hello @shahednasser ma'am,
I have made the suggested changes.
Could you please check once and let me know if I should make any more improvements to make it good?
Will be more than happy to learn in the process.
Thank you

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks @HVbajoria 🚀

**Comment by HVbajoria** [comment]
Thank you @shahednasser
